finance/ledger/Portfile

```diff
@@ -5,11 +5,16 @@ PortGroup           active_variants 1.1
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        ledger ledger 3.1.3 v
+github.setup        ledger ledger 3.2.1 v
+set major           3
+set minor           2
+set revision        1
 revision            1
```

Comment:
Remove the "set major", "set minor", "set revision" lines. Change the "revision 1" line to "revision 0".